cg/meta/upload/gt.py

```diff
@@ -126,3 +127,16 @@ def _is_variant_file(genotype_file: housekeeper_models.File):
 
     def _get_genotype_files(self, version_id: int) -> list:
         return self.hk.files(version=version_id, tags=["genotype"]).all()
+
+    @staticmethod
+    def genotype_check(case_obj: models.Family) -> bool:
+        """Check if balsamic case is contains WGS and normal sample"""
```

Comment:
```suggestion
        """Check if a cancer case is contains WGS and normal sample."""
```
Let´s not lock it to balsamic - it could for instance be RNAFusion in the future